src/modules/villages/village_handler.ts

```diff
@@ -1,14 +1,15 @@
 import express, { Request, Response, NextFunction } from 'express'
 import httpStatus from 'http-status'
+import { Village as Entity } from './village_entity'
 import { Village as Service } from './village_service'
 
 const router = express.Router()
 
 router.get(
   '/v1/villages/list-with-location',
-  async (req: Request, res: Response, next: NextFunction) => {
+  async (req: Request<never, never, never, Entity.RequestQuery>, res: Response, next: NextFunction) => {
     res.status(httpStatus.OK).json({
-      data: await Service.findAllWithLocation()
+      data: await Service.findAllWithLocation(req.query)
```

Comment:
#ASK

for format output json is any discussion before? 
this is my reading article maybe you can considering that ,, 😄 
[https://medium.com/@bojanmajed/standard-json-api-response-format-c6c1aabcaa6d](https://medium.com/@bojanmajed/standard-json-api-response-format-c6c1aabcaa6d)